apps/site/components/Common/ActiveLink/index.tsx

```diff
@@ -26,7 +27,9 @@ const ActiveLink: FC<ActiveLocalizedLinkProps> = ({
       ? // When using allowSubPath we want only to check if
         // the current pathname starts with the utmost upper level
         // of an href (e.g. /docs/...)
-        pathname.startsWith(`/${href.toString().split('/')[1]}`)
+        pathname.startsWith(`/${href.toString().split('/')[1]}`) &&
+        // but not when this link is for the deep link shortcut to previous releases
+        href.toString() !== VERSION_SUPPORT_SHORTCUT
```

Comment:
if this doesn't merge by morning, i will add unit tests for the entire `allowSubpath` part of the ternary, which has no test coverage. it requires mocking `usePathname`

Reply:
added via https://github.com/nodejs/nodejs.org/pull/7086/commits/9847aef066a21499174b84b59e8876945601195a